package/com.unity.formats.usd/Tests/Editor/ImportHelpersTests.cs

```diff
@@ -4,95 +4,127 @@
 using UnityEngine;
 using USD.NET;
 using USD.NET.Unity;
+using System.Linq;
 using Object = UnityEngine.Object;
+using System;
 
 namespace Unity.Formats.USD.Tests
 {
     public class ImportHelpersTests : BaseFixtureEditor
     {
-        Scene CreateTestAsset(string fileName)
+        [Test]
+        public void ImportAsPrefabTest_SceneClosedAfterImport()
+        {
+            var scene = CreateTestUsdScene();
+            ImportHelpers.ImportAsPrefab(scene, GetPrefabPath());
+            Assert.IsNull(scene.Stage, "Scene was not closed after import.");
+        }
+
+        [Test]
+        public void ImportAsTimelineClipTest_SceneClosedAfterImport()
+        {
+            var scene = CreateTestUsdScene();
+            ImportHelpers.ImportAsTimelineClip(scene, GetPrefabPath());
+            Assert.IsNull(scene.Stage, "Scene was not closed after import.");
+        }
+
+        [Test]
+        public void ImportAsPrefabTest_ImportClosedScene_Error()
+        {
+            var scene = CreateTestUsdScene();
+            scene.Close();
+            Assert.Throws<System.NullReferenceException>(() => ImportHelpers.ImportAsPrefab(scene, GetPrefabPath()));
+        }
+
+        [Test]
+        public void ImportAsTimelineClipTest_ImportClosedScene_Error()
         {
-            var dummyUsdPath = CreateTmpUsdFile(fileName);
-            var scene = ImportHelpers.InitForOpen(dummyUsdPath);
-            scene.Write("/root", new XformSample());
-            scene.Write("/root/sphere", new SphereSample());
-            scene.Save();
-            return scene;
+            var scene = CreateTestUsdScene();
+            scene.Close();
+            Assert.Throws<System.NullReferenceException>(() => ImportHelpers.ImportAsTimelineClip(scene, GetPrefabPath()));
         }
 
         [Test]
         public void ImportAsPrefabTest_ContentOk()
         {
-            var scene = CreateTestAsset("dummyUsd.usda");
+            var scene = CreateTestUsdScene();
             var assetPath = ImportHelpers.ImportAsPrefab(scene, GetPrefabPath());
-            Assert.IsNull(scene.Stage, "Scene was not closed after import.");
```

Comment:
Is there a reason not to keep this check?

Reply:
This particular part was tested already on the test case: ImportAsPrefabTest_SceneClosedAfterImport
No need to test it again in this test case - it would be redundant